kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConfig.java

```diff
@@ -109,6 +109,23 @@
       + "capture groups that are referenced in the format string using placeholders (i.e. $1) "
       + "(form of <topic regex>=<format string>)";
 
+  public static final String RECORD_ALIASES_CONFIG =                         "recordAliases";
+  private static final ConfigDef.Type RECORD_ALIASES_TYPE =                  ConfigDef.Type.LIST;
+  private static final ConfigDef.Importance RECORD_ALIASES_IMPORTANCE =      ConfigDef.Importance.MEDIUM;
+  private static final Object RECORD_ALIASES_DEFAULT =                        null;
+  private static final String RECORD_ALIASES_DOC =
+      "A list of mappings from record name regexes to table name postfixes. Note the regex must include "
```

Comment:
Why "postfixes"?

Reply:
BigQuery table name will follow the `<topic_table_name>_<record_alias>` template. Maybe I used incorrect wording, so I'm going to change the description and include the aforementioned template to make it clear.